src/routes/LoginEnterprise.tsx

```diff
@@ -52,19 +50,9 @@ export const validate = (values: IValues): IFormErrors => {
 };
 
 export const LoginEnterpriseRoute: FC = () => {
-  const {
-    accounts: { enterpriseAccounts },
-    loginEnterprise,
-  } = useContext(AppContext);
+  const { loginEnterprise } = useContext(AppContext);
   const navigate = useNavigate();
 
-  useEffect(() => {
-    if (enterpriseAccounts.length) {
```

Comment:
I love that we're extending this to support multiple enterprise accounts. Awesome!

This logic which limits Enterprise Accounts to at most 1 is here since there currently isn't a way to view all accounts, or edit/remove accounts.  

I have been dabbling on two similar changes

- View accounts: https://github.com/gitify-app/gitify/tree/feature/accounts-view
- Support multiple accounts (GitHub or GitHub Enterprise) - https://github.com/gitify-app/gitify/tree/refactor/accounts

My suggestion for this PR would be to defer this change until the above items are addressed